pennylane/ops/op_math/controlled.py

```diff
@@ -589,7 +594,7 @@ def pow(self, z):
             for op in base_pow
         ]
 
-    def simplify(self) -> "Controlled":
+    def simplify(self) -> "Operator":
```

Comment:
is this change because of the Identity corner-case handling you added?

Reply:
Yes exactly.